.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "external/aos_core_common_cpp"]
+	path = external/aos_core_common_cpp
+	url = https://github.com/MykolaSolyanko/aos_core_common_cpp.git
```

Comment:
Readme info how to build with submodules should be added?

Reply:
Done